rear_rider_device/cam_stream_unit_test.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import rear_rider_sensors.camera as camera
+import socket
+from time import sleep
+from unittest import IsolatedAsyncioTestCase
+
+class AsyncCameraUnitTest(IsolatedAsyncioTestCase):
+    
+    cam : camera.RRCamera = None
+
+    async def test_beginstream(self):
```

Comment:
Could you add a docstring to every test case so we know exactly what the function is testing? For instance, `self.assertTrue` is used but it would be nice to not have to consult the python documentation to understand what condition is being tested.